lib/vyasa_web/components/contexts/read/tracks.ex

```diff
@@ -14,38 +14,38 @@ defmodule VyasaWeb.Context.Read.Tracks do
   def render(assigns) do
     ~H"""
     <div>
+      <div>TRACKS component: shows single track list</div>
       <.table
         id="tracks"
         rows={@tracks}
         row_click={
           fn {_id, track} ->
             JS.push("navigate_from_track",
-              value: %{target: ~p"/explore/#{track.title}/"},
+              value: %{target: ~p"/explore/tracks/#{track.id}/"},
               target: @myself
             )
           end
         }
       >
         <:col :let={{_id, track}} label="">
-          <div class="font-dn text-2xl">
-            <%= to_title_case(track.title) %>
+          <div id={ "foo_track_" <> track.id } class="font-dn text-2xl">
+            TODO track foo_track_{track.id} view: <br /> {to_title_case(track.event.verse.body)}
```

Comment:
this was a stubbed example of deterministic ids that could be targetted by the event handler and emphasized

it was supposed to be an example to follow for the content_action = :show_verses

we can clean this up and it's good to go